cmake/developer_package/features.cmake

```diff
@@ -42,9 +42,13 @@ ie_option (BUILD_SHARED_LIBS "Build as a shared library" ON)
 
 ie_dependent_option (ENABLE_FASTER_BUILD "Enable build features (PCH, UNITY) to speed up build time" OFF "CMAKE_VERSION VERSION_GREATER_EQUAL 3.16" OFF)
 
-ie_dependent_option (ENABLE_CPPLINT "Enable cpplint checks during the build" ON "UNIX;NOT ANDROID" OFF)
+if(NOT DEFINED ENABLE_CPPLINT)
+	ie_dependent_option (ENABLE_CPPLINT "Enable cpplint checks during the build" ON "UNIX;NOT ANDROID" OFF)
```

Comment:
why not just to add `UNIX OR WINDOWS;NOT ANDROID`

Reply:
Could be done this way, but I have no knowledge about the build/test infrastructure and I'm not sure if it will handle the cpplint being enabled by default during builds on Windows as well. So I wanted to just let users enable it locally.